Jenkinsfile

```diff
@@ -26,7 +26,7 @@ node('build-slave') {
 
       stage('Unit Tests') {
 
-          sh "sudo mvn test '-Dtest=!%regex[io.opensaber.registry.client.*]'"
+          sh "sudo mvn test '-Dtest=!%regex[io.opensaber.registry.client.*]' -DfailIfNoTests=false"
```

Comment:
once tested locally - for multi-project - i.e. from lms-service - will be merging it in